sdc/rabbit/publisher.py

```diff
@@ -17,12 +17,13 @@ class QueuePublisher(object):
     """
     _durable_queue = True
 
-    def __init__(self, urls, queue, **kwargs):
+    def __init__(self, urls, queue, confirm_delivery=False, **kwargs):
```

Comment:
I would leave this out and just document the possibility of passing confirm_delivery as a kwarg, then at L39 do:

```python
if confirm_delivery:
    self._confirm_delivery = True
```